cmd/ebrelayer/txs/parser.go

```diff
@@ -68,14 +68,14 @@ func EthereumEventToEthBridgeClaim(valAddr sdk.ValAddress, event types.EthereumE
 	nonce := int(event.Nonce.Int64())
 
 	// Package the information in a unique EthBridgeClaim
-	witnessClaim.EthereumChainID = chainID
-	witnessClaim.BridgeContractAddress = bridgeContractAddress
-	witnessClaim.Nonce = nonce
-	witnessClaim.TokenContractAddress = tokenContractAddress
+	witnessClaim.EthereumChainId = int64(chainID)
```

Comment:
The old version of this EthBridgeClaim object had the chainid be an int. We should do the same thing to our protos so that we don't have to do this weird casting here.